pkg/ccl/changefeedccl/changefeed_dist.go

```diff
@@ -195,6 +200,21 @@ func fetchSpansForTables(
 		ctx, execCtx, details.Select, tableDescs[0], target, includeVirtual)
 }
 
+var replanChangefeedThreshold = settings.RegisterFloatSetting(
+	settings.TenantWritable,
+	"ccl.replan_flow_threshold",
+	"fraction of initial flow instances that would be added or updated above which a redistribution would occur (0=disabled)",
+	0.1,
+)
+
+var replanChangefeedFrequency = settings.RegisterDurationSetting(
+	settings.TenantWritable,
+	"ccl.replan_flow_frequency",
+	"frequency at which changefeed checks to see if redistributing would change its physical execution plan",
+	time.Minute/30,
```

Comment:
almost never see this written this way... perhaps `2 * time.Second`?
Regardless, this is way, way too frequent. 10 * time.Minute might be a more reasonable default.